build.gradle

```diff
@@ -404,6 +404,11 @@ project('standalone') {
         testCompile project(path:':segmentstore:server:host', configuration:'testRuntime')
     }
 
+    configurations {
+        runtime.exclude group: "com.sun.jersey", module: "jersey-core"
```

Comment:
I am assuming this does not impact the HDFS instance and related functionality that is being run as part of the standalone task. Ensure this is tested.